setup.md

```diff
@@ -174,44 +174,206 @@ before pushing them to GitHub.
-    [rbenv](https://github.com/rbenv/rbenv):
-
+### Windows
+
+#### Option 1 - Using the Windows Subsystem for Linux (WSL)
+
+If your version of Windows supports it, using the WSL will make the installation of the tools needed easier. Instructions to install Linux distributions from Windows 10/Windows Server are available from the [Microsoft website](https://docs.microsoft.com/en-us/windows/wsl/about).
+
+Once you have installed a Linux distribution, you can follow the installation instructions for [Linux](#linux-ubuntu) listed below. If you install a distribution other than Ubuntu, you'll need to adjust the commands that install the packages. 
+
+#### Option 2 - Using Windows
+
+With the instructions below, you'll be able to preview websites for non-R based lessons. You'll be able to do so from the Git Bash terminal or from the "Command Prompt with Ruby" that comes with the Ruby installation. You won't be able to use the `make` commands as Make isn't available from the Git Bash terminal (see the optional section below for more info).
+
+1. **Ruby**, go to <https://rubyinstaller.org/> choose Ruby+DevKit for your
+   system (typically x64), and the most recent version available. During the
+   installation process, choose to install the MSYS2 toolchain. On the last step
+   of the installation screen, make sure that "Run 'ridk install'" is checked.
+   This will bring a terminal window with 3 options, press "Enter" (for the
+   default installation). After this step completes, you'll be prompted again,
+   and press "Enter" again (for the default option). Once the installation is
+   complete, restart your system.
+
+2. Navigate to the folder that contains the lesson, and you can now use `bundle
+   exec jekyll serve` from your Git Bash terminal to preview the lessons.
+
+##### Optional (but recommended)
+
+With these instructions, you'll be able to use the `make` commands and render
+all lessons including those that use R. However, you won't be able to do so from the Git Bash terminal, but from the other terminals (Windows Powershell, cmd.exe, or the Command Prompt with Ruby).
+
+1. In the File Explorer, right-click on "This PC" icon, and click on
+   "Properties". Click on "Advanced System Settings", and click on the button
+   "Environment Variables". Click on the variable "Path", and then the "Edit"
+   button. Click on the "New" button and add `C:\Ruby26-x64\msys64\usr\bin` (use
+   the File Explorer to make sure this is the correct path for your Ruby and
+   MSYS2 installation). If you're working on R-based lessons and R isn't already
+   listed there, you need to add it by adding `C:\Program Files\R\R-3.x.x\bin`
+   (using the correct path and R version number for your installation).
+   
+2. Open the "Command Prompt with Ruby" (if it was open before you edited the
+   Path, close it and re-open it).
+   
+3. Go to the section [For Everyone](#for-everyone)
+
+### macOS
+
+1. First make sure you have Homebrew installed by doing
```

Comment:
note that Homebrew is also referred to as brew here before referring to it as brew

Reply:
The right fix is to change "brew" below to Homebrew. `brew` on its own is a command that Homebrew provides.